f5_openstack_agent/lbaasv2/drivers/bigip/test/test_agent_manager.py

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-from f5_openstack_agent.lbaasv2.drivers.bigip import agent_manager
+from .. import agent_manager
```

Comment:
Why are you using dots in some cases and removing them in others?

Why not just set up a blanket, consistent approach at treating all code the same?

Reply:
All "system" packages, i.e. stdlib, and pip install-requires, e.g.:   `os`, `random`, `neutron` code shall be imported absolutely, all _within package_ code shall be imported relatively.

This sets a clear boundary between the two kinds of code:

(0) system code that is not part of this package
(1)  source code that _is_ part of this package

This enhances readability, additionally it makes it within-package imports (the ones we specify relatively, function *without respect to* whether the package has been installed).